docs/spock_primer.adoc

```diff
@@ -540,6 +542,8 @@ with(service) {
   1 * stop()
 }
 ----
+
+[[specifications_as_documentation]]
```

Comment:
This will conflict with your other #787  please align them

Reply:
Added these lines to this file in #787 per request.